aten/src/ATen/native/native_functions.yaml

```diff
@@ -561,9 +561,9 @@
 
 - func: embedding_dense_backward(Tensor grad, IndexTensor indices, int64_t num_weights, int64_t padding_idx, bool scale_grad_by_freq) -> Tensor
   variants: function
-  dispatch:
-    CPU: embedding_dense_backward_cpu
-    CUDA: embedding_dense_backward_cuda
+#  dispatch:
+#    CPU: embedding_dense_backward_cpu
+#    CUDA: embedding_dense_backward_cuda
```

Comment:
Oh oh. I don't think you can disable the CUDA variant. Sorry, I should have seen this earlier myself, too.